pandas/core/generic.py

```diff
@@ -2238,15 +2240,12 @@ def to_json(
         Encoding with Table Schema
 
         >>> df.to_json(orient='table')
-        '{"schema": {"fields": [{"name": "index", "type": "string"},
-                                {"name": "col 1", "type": "string"},
-                                {"name": "col 2", "type": "string"}],
-                     "primaryKey": "index",
-                     "pandas_version": "0.20.0"},
-          "data": [{"index": "row 1", "col 1": "a", "col 2": "b"},
-                   {"index": "row 2", "col 1": "c", "col 2": "d"}]}'
+        '{"schema":{"fields":[{"name":"index","type":"string"},\
+{"name":"col 1","type":"string"},{"name":"col 2","type":"string"}],\
```

Comment:
can you use ... here? (else could do a json prettify, e.g. json.dump(...., indent=4)